src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FILE_NOT_WRITTEN_ERROR_MESSAGE = "Can't write data to the file: ";
+
+    @Override
+    public void write(String data, String toFileName, String path) {
+        try (BufferedWriter bufferedReader = new BufferedWriter(
```

Comment:
The variable name 'bufferedReader' is misleading because it is actually a BufferedWriter. Please rename it to 'bufferedWriter' to accurately reflect its purpose.

Reply:
++